<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADFG(V)X</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="T_sifrovat">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sifrovat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="T_desifrovat">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Desifrovat</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="vstup">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="T_ukoncit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ukoncit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="vystup">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Vstupni text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Vystupni text</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="vypis_tabulky">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>761</width>
      <height>321</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>111</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>15</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>37</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabulka</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="perm_klic">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Zadej permutacni klic</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="subs_klic">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zadej substitucni klic&lt;/p&gt;&lt;p&gt;nebo vsechny znaky sifrovaci matice&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="vyber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ADFGX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ADFGVX</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="zmena">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nastavit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nejprve nastav mod</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>780</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Sifrovaci klic: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tab_text">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>780</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>810</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>(Potrebny k desifrovani)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>432</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Po spusteni aplikace je dulezite nastavit MOD sifrovani&lt;/p&gt;&lt;p&gt;Sifrovani&lt;/p&gt;&lt;p&gt;1. Vlozte text k sifrovani&lt;/p&gt;&lt;p&gt;2. Vlozte substitucni klic a nebo rovnou vsechchny znaky &lt;/p&gt;&lt;p&gt;sifrovaci matice&lt;/p&gt;&lt;p&gt;(25 znaku u ADFGX nebo 36 znaku u ADFGVX )&lt;/p&gt;&lt;p&gt;3. Vlozte permutacni klic&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Desifrovani&lt;/p&gt;&lt;p&gt;1. Vlozte text k desifrovani&lt;/p&gt;&lt;p&gt;2. Vlozte cely sifrovaci klic do kolonky dole&lt;/p&gt;&lt;p&gt;(25 znaku u ADFGX nebo 36 znaku u ADFGVX )&lt;/p&gt;&lt;p&gt;3. Vlozte permutacni klic&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
